tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
Why did this test need to be removed? It should still be possible to represent recursive relationships with the datasources JSON.